django/forms/models.py

```diff
@@ -279,6 +279,13 @@ def __new__(mcs, name, bases, attrs):
                 # fields from the model"
                 opts.fields = None
 
+            if opts.exclude:
```

Comment:
Interresting, where is that ?

I thought it had to be done this way because base_fields is inherited so it has already been setup by the parent class.